x-pack/plugins/security_solution/public/management/links.ts

```diff
@@ -232,23 +232,25 @@ export const getManagementFilteredLinks = async (
 ): Promise<LinkItem> => {
   try {
     const currentUserResponse = await plugins.security.authc.getCurrentUser();
-    const privileges = calculateEndpointAuthz(
-      licenseService,
-      plugins.fleet?.authz,
-      currentUserResponse.roles
-    );
-    if (!privileges.canAccessEndpointManagement) {
+    const { canAccessEndpointManagement, canIsolateHost, canAccessResponseActionsHistory } =
+      calculateEndpointAuthz(licenseService, plugins.fleet?.authz, currentUserResponse.roles);
+    if (!canAccessEndpointManagement) {
       return excludeLinks([
         SecurityPageName.hostIsolationExceptions,
         SecurityPageName.responseActionsHistory,
       ]);
     }
-    if (!privileges.canIsolateHost || !privileges.canAccessResponseActionsHistory) {
+
+    if (!canAccessResponseActionsHistory) {
```

Comment:
We also need to do the `summaryResponse.total` check in this case in the event that we're below `platinum`

Reply:
good catch 👍 wIll add the condition

Reply:
We will need to check if `canIsolateHost` too.  Something like `!summaryResponse.total && !canIsolateHost` for this condition

Reply:
5566a92a7369191bee80875fa4fcc1eb9f6e3d1e